src/frontend/src/pages/ProjectDetailPage/ProjectEdit/ProjectEditContainer.tsx

```diff
@@ -152,92 +152,93 @@ const ProjectEditContainer: React.FC<ProjectEditContainerProps> = ({ project, ex
   };
 
   return (
-    <form
-      id="project-edit-form"
-      onSubmit={(e) => {
-        e.preventDefault();
-        e.stopPropagation();
-        handleSubmit(onSubmit)(e);
-      }}
-      onKeyPress={(e) => {
-        e.key === 'Enter' && e.preventDefault();
-      }}
+    <PageLayout
+      title={`${wbsPipe(project.wbsNum)} - ${project.name}`}
+      previousPages={[{ name: 'Projects', route: routes.PROJECTS }]}
+      actionButton={
```

Comment:
this prop should be renamed since we're using for stuff other than the action button. Maybe titleAction, or rightItem or smthn (don't love those, if you can come up with a better name use that)